### PR TITLE
Correction of labeler

### PR DESCRIPTION
**Pull request by tomkocmi**
When I removed the tanh, the performance on both CNN lemmatizer and RNN tagger raised.

### CODE REVIEW

**Comment by tomkocmi** [comment]
If it sounds weird I could do some thorough tests for you. But as a rule of thumbs, there should never be a nonlinear layer on the output layer. And if you check how the logits are handeled you find out the nonlinearity only brings problems.
 - sparse_softmax_cross_entropy_with_logits do not care it if is normalized (more over it should not be normalized by softmax)
 - argmax do not care about the nonlinearity
 - softmax is by my opinion the reason for worse performance with the nonlinearity.
